src/rpc/client.cpp

```diff
@@ -116,16 +116,18 @@ static const CRPCConvertParam vRPCConvertParams[] =
     { "zcrawjoinsplit", 4 },
     { "zcbenchmark", 1 },
     { "zcbenchmark", 2 },
-    { "getblocksubsidy", 0 },
-    { "z_listreceivedbyaddress", 1 },
-    { "z_getbalance", 1 },
-    { "z_gettotalbalance", 0 },
-    { "z_gettotalbalance", 1 },
-    { "z_sendmany", 1 },
-    { "z_sendmany", 2 },
-    { "z_sendmany", 3 },
-    { "z_sendmany", 4 },
-    { "sc_sendmany", 0 },
+    { "getblocksubsidy", 0},
+	{ "getblockmerkleroots", 0},
+	{ "getblockmerkleroots", 1},
+    { "z_listreceivedbyaddress", 1},
+    { "z_getbalance", 1},
+    { "z_gettotalbalance", 0},
+    { "z_gettotalbalance", 1},
```

Comment:
```suggestion
    { "getblocksubsidy", 0 },
    { "getblockmerkleroots", 0 },
    { "getblockmerkleroots", 1 },
    { "z_listreceivedbyaddress", 1 },
    { "z_getbalance", 1 },
    { "z_gettotalbalance", 0 },
    { "z_gettotalbalance", 1 },
```
NIT consistent formatting